.gitignore

```diff
@@ -5,3 +5,6 @@ package-lock.json
 
 package.json
 
+
+
```

Comment:
Kindly add `node_modules` to the `.gitignore` file as it's in the requirement of the project in any project that uses linters repo so if you set up linters locally the node_modules folder is not uploaded.